CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v4.0.0](https://github.com/puppetlabs/rspec-puppet/tree/v4.0.0) - 2023-10-09
+
+[Full Changelog](https://github.com/puppetlabs/rspec-puppet/compare/v3.0.0...v4.0.0)
+
+### Changed
+- (CAT-1226) - Remove Compatibility for Puppet 7.11 and below [#73](https://github.com/puppetlabs/rspec-puppet/pull/73) ([jordanbreen28](https://github.com/jordanbreen28))
```

Comment:
Is that correct? Doesn't it drop support for 7.10 and older?

Reply:
yes you are spot on.. thank you for noticing this!